crates/swift-bridge-ir/src/bridged_type/shared_enum.rs

```diff
@@ -53,6 +53,24 @@ impl SharedEnum {
         quote! { #name }
     }
 
+    pub fn ffi_union_name_string(&self) -> String {
+        format!("{}Fields", self.ffi_name_string())
+    }
+
+    pub fn ffi_union_field_names_string(&self) -> String {
```

Comment:
Can you add a doc comment with an example of a string that this might create? 
(Like the other methods in this file)

This makes it easier for new contributors to more quickly understand what these utility methods are doing.